### PR TITLE
Fixed the full height but in liveboard in  multitab

### PR DESCRIPTION
**Pull request by suhel-spot**

### CODE REVIEW

**Review by Nirmay15 — COMMENTED** [review]
Lets add UT for this

File: src/embed/liveboard.spec.ts [hunk 1]
```diff
@@ -269,6 +270,62 @@ describe('Liveboard/viz embed tests', () => {
...
+            // Expose the protected method for testing
+            constructor(domSelector: DOMSelector, viewConfig: LiveboardViewConfig) {
+                viewConfig.embedComponentType = 'LiveboardEmbed';
+                super(domSelector, viewConfig);
+            }
+
+            public exposeSetIFrameHeight(value = 500) {
+                super.setIFrameHeight(value);
+            }
+        }
```
Comment (Nirmay15) [inline src/embed/liveboard.spec.ts:285]: why are you mocking the class, you can directly test on our embed classes. This test is not checking fullheight I think you can mock on function and check right? Constructor already listens on this event.

File: src/embed/liveboard.spec.ts [hunk 2]
```diff
@@ -269,6 +270,62 @@ describe('Liveboard/viz embed tests', () => {
...
```
Comment (Nirmay15) [inline src/embed/liveboard.spec.ts:313]: Same as above

File: src/embed/liveboard.spec.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ import {
     Action,
     AuthType,
     ContextMenuTriggerOptions,
+    DOMSelector,
```
Comment (Nirmay15) [inline src/embed/liveboard.spec.ts:7]: Not required remove it

**Review by Nirmay15 — APPROVED** [review]

File: src/embed/liveboard.spec.ts (hunk 3 above)
Comment (suhel-spot) [reply src/embed/liveboard.spec.ts]: removed

**Pull request merged**